vision/ui/main_window.py

```diff
@@ -0,0 +1,67 @@
+from tkinter import*
+ 
+# creating button for autonomous mapping
+window = Tk()
```

Comment:
You can put all of this code under one method called `run(stream)` so that other code can call it.